src/model/inventory/InventoryOutput.java

```diff
@@ -0,0 +1,13 @@
+package model.inventory;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class wil be used to create documents that remove stock to the Inventory
```

Comment:
Could you ellaborate a bit more on what is this for?

Please correct the typos, use google translate if necessary.